app/components/FileLoader.js

```diff
@@ -85,11 +97,6 @@ export default class FileLoader extends Component {
       y: this.refTableScroll.scrollTop,
     });
 
-    this.props.storeFileLoadState({
-      filesToRender: this.state.filesToRender,
```

Comment:
This appears to be left over to when the filesToRender and filesOffset were stored in component state rather than redux. This is no longer needed and causes problems because the values are set to undefined.